README.md

```diff
@@ -180,7 +182,7 @@ Additionally, if image_url is not supplied neither Trivy nor Binary Attestation
 - Runs Trivy, a comprehensive security scanner. Does not run on draft pull requests.
 - Creates a binary attestation on the supplied image if trivy is run.
 - Calls the Github Security Code Scanning API and fails with exit code 1 if there are any _high_ or _critical_ errors.
-- Creates a binary attestation on the supplied image if both TFsec and Trivy scans are run and there are no _high_ or _critical_ errors in Github Security Code Scanning.
+- The above step can be overriden using the `allow_severity_level` input.
```

Comment:
```suggestion
- The above step can be configured by specifying the `allow_severity_level` parameter.
```

Jeg mener "overridden" gir litt andre assosiasjoner. Dette steget er mandatory da tilstedeværelsen av en slik attestering er påkrevet :)

Reply:
Fair, but GitHub uses "input" as the name for things you send into GitHub workflows, so I'd prefer the use of that word :)

Reply:
Smooth - i changed it to "input parameter". I believe most devs will be familiar with the term:)